common/scala/src/main/resources/reference.conf

```diff
@@ -1,5 +1,6 @@
-whisk.spi{
+whisk.spi {
   ArtifactStoreProvider = whisk.core.database.CouchDbStoreProvider
   MessagingProvider = whisk.connector.kafka.KafkaMessagingProvider
   ContainerFactoryProvider = whisk.core.containerpool.docker.DockerContainerFactoryProvider
+  #ContainerFactoryProvider = whisk.core.mesos.MesosContainerFactoryProvider
```

Comment:
Would this be commented out but provided as an example?

Will be test this with Travis? Jenskins?

Reply:
yes, an example;

testing can be done to the extent of making sure the mesos-actor receives messages that are expected, but I wouldn't expect to run a mesos cluster; we need to trust the the mesos-actor maintains the support of the mesos API; additional tests will be added to mesos-actor as well.